central/administration/events/datastore/types.go

```diff
@@ -0,0 +1,4 @@
+package datastore
+
+// The timestamp format / layout is borrowed from `pkg/search/postgres/query/time_query.go`.
+const TimestampLayout = "01/02/2006 3:04:05 PM MST"
```

Comment:
Where it's used?

Reply:
It's not, I still had it from an earlier version before refactoring, I've removed it now 👍